<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PointSelect</class>
 <widget class="QWidget" name="PointSelect">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>973</width>
    <height>329</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="staticLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Current value:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="QLabel" name="valueLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>None</string>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="questionLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>225</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Please enter the coordinates:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label_1">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>25</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>25</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>   x</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QDoubleSpinBox" name="doubleSpinBox_1">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="keyboardTracking">
        <bool>false</bool>
       </property>
       <property name="minimum">
        <double>-1000000000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1000000000.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLabel" name="label_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>25</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>25</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>   y</string>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QDoubleSpinBox" name="doubleSpinBox_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="keyboardTracking">
        <bool>false</bool>
       </property>
       <property name="minimum">
        <double>-1000000000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1000000000.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="1" column="5">
      <widget class="QLabel" name="label_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>25</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>25</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>   z</string>
       </property>
      </widget>
     </item>
     <item row="1" column="6">
      <widget class="QDoubleSpinBox" name="doubleSpinBox_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="keyboardTracking">
        <bool>false</bool>
       </property>
       <property name="minimum">
        <double>-1000000000.000000000000000</double>
       </property>
       <property name="value">
        <double>0.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="1" column="7">
      <widget class="QLabel" name="unitsLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>(unit)</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="8">
      <widget class="QCheckBox" name="checkBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Enable z-coordinate</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>725</width>
       <height>188</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>doubleSpinBox_1</tabstop>
  <tabstop>doubleSpinBox_2</tabstop>
  <tabstop>doubleSpinBox_3</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
